src/main/java/org/osiam/AuthServer.java

```diff
@@ -0,0 +1,93 @@
+package org.osiam;
+
+import com.google.common.collect.ImmutableMap;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.flywaydb.core.Flyway;
+import org.flywaydb.core.api.MigrationVersion;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.context.web.SpringBootServletInitializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.filter.CharacterEncodingFilter;
+
+import javax.servlet.Filter;
+import javax.sql.DataSource;
+import java.util.Map;
+
+@SpringBootApplication
+@EnableTransactionManagement
+@EnableAspectJAutoProxy(proxyTargetClass = true)
```

Comment:
Yes, everything fails otherwise. I can't remember exactly why, but it has something to do that we don't have interfaces for some service classes so that normal JDK proxies don't work.